internal/functions/serve/templates/main.ts

```diff
@@ -91,7 +100,7 @@ serve(async (req: Request) => {
   console.error(`serving the request with ${servicePath}`);
 
   const memoryLimitMb = 150;
-  const workerTimeoutMs = 1 * 60 * 5000;
+  const workerTimeoutMs = 1 * 60 * 1000;
```

Comment:
https://github.com/supabase/cli/pull/1138

Could you explain the decision to change back to 1 min? Is this linked issue no longer valid?

Reply:
@mansueli what was the context for the change in #1138? Was it any particular function having timeouts?

I saw the 5000, thought it was a typo. Can change it back (to 5 * 60 * 1000)

Reply:
It's explained in [this issue](https://github.com/supabase/cli/issues/1137) which is closed by that PR.